packages/lodestar/src/network/gossip/gossip.ts

```diff
@@ -147,6 +147,7 @@ export class Gossip extends (EventEmitter as { new(): GossipEventEmitter }) impl
     event: keyof IGossipEvents | string,
     listener?: unknown,
     params: Map<string, string> = new Map()): void {
+    if (!this.listeners(event as keyof IGossipEvents).includes(listener as (...args: unknown[]) => void)) return;
```

Comment:
@wemeetagain so gossip is fine, this caused the incorrect unsubscription of gossip topic